pkgs/tools/system/bashtop/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, bash_5
+, curl
+, lm_sensors
+, procps
+, sysstat
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bashtop";
+  version = "0.8.23";
+
+  src = fetchFromGitHub {
+    owner = "aristocratos";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0y6yxm2vmbz0373cfdl6mjh8vhs0r0wcng82n304klx90qxg3ljp";
+  };
+
+  buildInputs = [ bash_5 curl lm_sensors procps-ng sysstat ];
```

```suggestion
  buildInputs = [ bash_5 curl lm_sensors procps sysstat ];
```

Sorry, forgot to suggest this change, too.

---

And I forgot to try building it. Thanks!